src/boardTest.java

```diff
@@ -0,0 +1,234 @@
+package Java;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.junit.jupiter.api.Test;
+
+public class boardTest {
+	
+	@Test public void test_01() {
+
+		//One player Moving into a building and exiting
+		ArrayList<Player> players = new ArrayList<Player>();
+		Player p = new Player("Simon", true);

```

Comment:
Out of date creation of player, needs to be Player(String, List<Cards>, Character)